openmls/src/schedule/mod.rs

```diff
@@ -745,6 +757,57 @@ impl ExporterSecret {
     }
 }
 
+/// A secret that we can derive secrets from, that are used outside of OpenMLS.
+/// In contrast to `[ExporterSecret]`, the `[ApplicationExportSecret]` is not
+/// persisted. It can be deleted after use to achieve forward secrecy.
+#[cfg(feature = "extensions-draft")]
+#[derive(Debug, Serialize, Deserialize, Clone)]
```

Comment:
Don't derive `Debug`. We shouldn't allow printing secrets. Or only with the crypto debug feature.

Reply:
Since all it contains is `Secret`, which already redacts its content, we should be fine here.